src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -300,12 +300,16 @@ def set(self, bucket, bucket_id, uid, versioning_state=None,
         uid_tenant = uid[:uid.find('$')] if uid.find('$') >= 0 else None
         bucket_name = RgwBucket.get_s3_bucket_name(bucket, uid_tenant)
 
+        # Update locking if it is enabled.
```

Comment:
Provided the code that this comment is talking about is 10 lines below why not just putting the 
```
if locking['lock_enabled']:
            self._set_locking(uid, daemon_name, bucket_name, lock_mode,
                              lock_retention_period_days,
```
logic just before ``versioning_state``` code so the comment and the code are together?

Reply:
I instead moved the comment below.